### PR TITLE
Anti pearl glitch

### PR DESCRIPTION
**Pull request by SaithTime**

### CODE REVIEW

**Review by CyberFlameGO — APPROVED** [review]
LGTM

**Review by xzolw — APPROVED** [review]

**Review by sadcenter — APPROVED** [review]
very useful IMO

File: NachoSpigot-Server/src/main/java/net/minecraft/server/EntityEnderPearl.java [hunk 1]
```diff
@@ -78,6 +78,32 @@ protected void a(MovingObjectPosition movingobjectposition) {
                     location.setPitch(player.getLocation().getPitch());
                     location.setYaw(player.getLocation().getYaw());
 
+                    // Nacho start - Anti ender pearl glitch
+
+                    if (Nacho.get().getConfig().antiEnderPearlPatch) {
```
Comment (Lucaskyy) [inline NachoSpigot-Server/src/main/java/net/minecraft/server/EntityEnderPearl.java:83]: Could you please fix the merge conflicts?

**Review by Lucaskyy — CHANGES_REQUESTED** [review]
LGTM, but please fix the merge conflicts!

File: NachoSpigot-Server/src/main/java/net/minecraft/server/EntityEnderPearl.java (hunk 1 above)
Comment (SaithTime) [reply NachoSpigot-Server/src/main/java/net/minecraft/server/EntityEnderPearl.java]: If it was just the configuration I removed it for the moment

**Comment by CyberFlameGO** [comment]
@Lucaskyy for in future, usually pull requests have an option enabled by default, which allows a maintainer to commit/push to the branch being PR'd. I also feel that merging this first would've been smarter
@SaithTime also it seems like there are still conflicts.

**Comment by Lucaskyy** [comment]
> for in future, usually pull requests have an option enabled by default, which allows a maintainer to commit/push to the branch being PR'd.

@CyberFlameGO I know, I just don't have the time at the moment to do it myself.

**Comment by CyberFlameGO** [comment]
Alright, fair enough

**Comment by Lucaskyy** [comment]
Superseded by #222

**Pull request closed without merging**